pagopa/io-app
# [IOBP-307, IOBP-308] Add payment notice number and fiscal code new input screens

## Short description
This PR adds two new screens for the pagoPA payment notice manual input flow: `WalletPaymentInputNoticeNumberScreen` and `WalletPaymentInputFiscalCodeScreen`.

https://github.com/pagopa/io-app/assets/6160324/fc771200-beb6-4fa9-9281-54c32bc0cfcc

## List of changes proposed in this pull request
- New screen: `WalletPaymentInputNoticeNumberScreen`
- New screen: `WalletPaymentInputFiscalCodeScreen`
- Removed deprecated navigation actions
- Added required locales keys

## How to test
Enable the experimental design system from the **Profile screen -> Experimental Design System** toggle.
You can view the manual input flow from either the home scan button or from the "Pay notice" button in the wallet section.
